commons-geometry-spherical/src/test/java/org/apache/commons/geometry/spherical/twod/GreatCircleTest.java

```diff
@@ -664,7 +664,7 @@ void testHashCode() {
         Assertions.assertEquals(hash, a.hashCode());
 
         Assertions.assertNotEquals(hash, b.hashCode());
-        Assertions.assertNotEquals(hash, c.hashCode());
+        Assertions.assertEquals(hash, c.hashCode());
```

Comment:
If this is testing the hash codes are equal (because the objects are equal) then we should test the objects are equal:
```java
// Equal objects have equal hash codes
Assertions.assertEquals(a, c);
Assertions.assertEquals(hash, c.hashCode());
```